strawberryfields/ops.py

```diff
@@ -911,11 +911,39 @@ class GKP(Preparation):
     """
 
     def __init__(
-        self, state=None, epsilon=0.2, cutoff=1e-12, representation="real", shape="square"
+        self, state=None, epsilon=0.2, ampl_cutoff=1e-12, representation="real", shape="square"
```

Comment:
You may want to change ``cutoff`` to ``ampl_cutoff`` in the ``Catstate`` preparation class too.